### PR TITLE
bug fixes and legacy cleanup

### PR DESCRIPTION
**Pull request by Abeous**
* fixed context menu overflow mentioned in #228 https://i.imgur.com/WZZmExI.png 
    * now shows upward when opened in lower third of chat
* added custom tag info into tag commands descriptions
* commented out /stalk and /mentions
  * commands can still be entered but will not show up in autocomplete or /help
* added context menu information to hints
* added custom tag info to hints

### CODE REVIEW

File: assets/chat/js/chat.js [hunk 1]
```diff
@@ -172,14 +169,15 @@ const commandsinfo = new Map([
...
     ["timestampformat", { desc: "Set the time format of the chat." }],
-    ["stalk", { desc: "Return a list of messages from <nick>", alias: ["s"] }],
-    [
-        "mentions",
-        {
-            desc: "Return a list of messages where <nick> is mentioned",
-            alias: ["m"]
-        }
-    ],
+    // Disabling /stalk and /mentions until functionality is added.
```
Comment (jbpratt78) [inline assets/chat/js/chat.js:172]: Go ahead and delete this stuff since it’s deleted above

File: assets/chat/js/chat.js [hunk 2]
```diff
@@ -1826,11 +1824,11 @@ class Chat {
                 MessageBuilder.info(
                     `Tagged nicks: ${[...this.taggednicks.keys()].join(
                         ","
-                    )}. Available colors: ${tagcolors.join(",")}`
+                    )}. Preset colors: ${tagcolors.join(",")}, or /tag user #HEXCODE`
                 ).into(this);
             } else {
                 MessageBuilder.info(
-                    `No tagged nicks. Available colors: ${tagcolors.join(",")}`
+                    `No tagged nicks. Preset colors: ${tagcolors.join(",")}, or /tag user #HEXCODE`
```
Comment (jbpratt78) [inline assets/chat/js/chat.js:1831]: Can these be pulled out as it looks like this is below as well

File: assets/chat/js/chat.js [hunk 3]
```diff
@@ -1882,11 +1880,11 @@ class Chat {
                 MessageBuilder.info(
                     `Tagged nicks: ${[...this.taggednicks.keys()].join(
                         ","
-                    )}. Available colors: ${tagcolors.join(",")}`
+                    )}. Preset colors: ${tagcolors.join(",")}, or \`/tag user #HEXCODE\``
```
Comment (jbpratt78) [inline assets/chat/js/chat.js:1883]: This as well

**Comment by Abeous** [comment]
Went ahead and gutted /stalk and /mentions command functions as well. I looked around and didn't find many other commands that warranted a global variable for command strings, so I added constants within the functions to clean it up a bit.

**Pull request merged**